components/Cards/SchoolCard/FrontSchoolCard.css

```diff
@@ -0,0 +1,65 @@
+.interactionsContainer {
+  align-items: center;
+  font-family: var(--primaryFontFamily), sans-serif;
```

Comment:
I meant line 3 of a different file `SchoolCard/SchoolCard.css`

Basically, that should make the font choice for everything in the card.